client/my-sites/site-settings/form-writing.jsx

```diff
@@ -73,20 +72,6 @@ class SiteSettingsFormWriting extends Component {
 				className="site-settings__general-settings"
 			>
 
-				{
-					this.props.isJetpackSite && this.props.jetpackMasterbarSupported && (
```

Comment:
Logic that sets `jetpackMasterbarSupported` prop can be removed now from this file, but should be added to the another one.